packages/location/src/LocationEditor.tsx

```diff
@@ -168,16 +171,18 @@ export function LocationEditorConnected(props: LocationEditorConnectedProps) {
     >
       {({ value, disabled, setValue, externalReset }) => {
         return (
-          <LocationEditor
-            // on external change reset component completely and init with initial value again
-            key={`location-editor-${externalReset}`}
-            value={value}
-            disabled={disabled}
-            setValue={setValue}
-            googleMapsKey={googleMapsKey}
-            selectedView={selectedView}
-            setSelectedView={setSelectedView}
-          />
+          <>
```

Comment:
any reason why we are now returning a fragment instead of just the location editor?

Reply:
hahaha damn, was testing stuff and it was left over. I've removed thanks for spotting!